jerry-core/ecma/operations/ecma-function-object.c

```diff
@@ -155,7 +156,7 @@ ecma_op_is_callable (ecma_value_t value) /**< ecma value */
  * @return ECMA_IS_VALID_CONSTRUCTOR - if object is a valid for constructor call
  *         any other value - if object is not a valid constructor, the pointer contains the error message.
  */
```

Comment:
The "any other" value" comment is now invalid, no pointer is returned here. Please update.